ensembl_genes/Red.py

```diff
@@ -0,0 +1,261 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Module to run Red to find repeats and store them in the given Ensembl core database """
+
+import os
+import errno
+import subprocess
+import tempfile
+import shutil
+import sqlalchemy as db
+from pathlib import Path,PurePath
+
+# sqlalchemy requires MySQLdb but MySQLdb doesn't support Python 3.x
+# pymysql can be imported and used instead
```

Comment:
I think sqlalchemy should work with any of the 3 currently maintained MySQL adapters [mentioned here](https://stackoverflow.com/a/46396881/4651668).

I've been querying the public ensembl MySQL instance using the "official" mysql-connector-python, which is now on [PyPI](https://pypi.org/project/mysql-connector-python/) and [conda](https://anaconda.org/conda-forge/mysql-connector-python), using:

```python
connection = "mysql+mysqlconnector://anonymous@ensembldb.ensembl.org/homo_sapiens_core_102_38"
```

Anyways, not sure if this is helpful but ran into a few errors like `sqlalchemy.exc.NoSuchModuleError` until settling on this configuration.

Reply:
Thanks Daniel. We'll review and probably change the mysql connectors in future when we add some more Python modules.